TW-Calc.user.js

```diff
@@ -1516,7 +1495,7 @@ window.TWCalc_inject = function () {
                     
                     TW_Calc.Craft.window.progressBar = new west.gui.Progressbar()
                         .setValue(Character.professionSkill)
-                        .setMaxValue(850);
+                        .setMaxValue(826);
```

Comment:
Why the new maximum value is 826?

Reply:
The recipes currently only go to 825. So the reachable maximum ist 826 right now.

Reply:
![image](https://user-images.githubusercontent.com/11925394/69010605-9f75da00-0961-11ea-8094-b4202c88e060.png)

The in-game window show 850 as a maximum. I would like to keep it the same as it is in the original game window.

Reply:
I know, I just thought, we could fix Inno's mistakes
But alright, I re-changed it.